cmd/soroban-cli/src/commands/contract/build.rs

```diff
@@ -82,8 +96,17 @@ pub enum Error {
     CopyingWasmFile(io::Error),
     #[error("getting the current directory: {0}")]
     GettingCurrentDir(io::Error),
+    #[error("reading wasm file: {0}")]
+    ReadingWasmFile(io::Error),
+    #[error("writing wasm file: {0}")]
+    WritingWasmFile(io::Error),
+    #[error("invalid metadata entry: {0}")]
+    MetadataArg(String),
```

Comment:
Prefer the term "meta" over "metadata" for consistency with existing vocab. Same elsewhere in the change where "metadata" is being used to refer to the contract meta.
```suggestion
    #[error("invalid meta entry: {0}")]
    MetaArg(String),
```

Reply:
good catch, i missed that one!